liaisons.md

```diff
@@ -59,5 +66,40 @@ of SIGs, WGs and UGs.
 | [Committee Code of Conduct](committee-code-of-conduct/README.md) | Tim Pepper (**[@tpepper](https://github.com/tpepper)**) |
 | [Committee Security Response](committee-security-response/README.md) | Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**) |
 <!-- BEGIN CUSTOM CONTENT -->
+## Expectations
+### Liaison will: 
+- Alert leads the Annual Report process is starting, expectations, and timeline
+- Check-ins with their groups at least once a quarter. This could mean a group DM
+with Chairs, a visit to a group meeting, or some other connection. Get a pulse 
+on health of the following:
+  - Contributor base
+    - Chairs and other leads: How are they?
+    - Contributors, reviewers, approvers: How are they?
+    - Newcomers: able to efficiently join and contribute?
+  - Operational Health
+    - Transparency of meetings and decision processes (eg: published meeting 
+  agendas, minutes, and video recordings)
+    - Communication best practices for shared status and decision making across 
+  a WG and its sponsor SIG(s) and a SIG and its subprojects
+    - Provide consultation on how to improve operational health
+  - Gap Analysis
+    - Where is the group now and where do they want/need to be?
+- Act as conduit for incoming cross-cutting deliverables requirements, such as 
+ones relative to test, infra, org, docs, etc.
+- Check that sustainability is a priority. 
+  - As contributors and maintainers come and go, what is the group doing to 
```

Comment:
Super minor nit: split this bullet of three question prompts into three bullets.